github-orgs/kubeflow/org.yaml

```diff
@@ -1213,12 +1190,13 @@ orgs:
           wg-pipeline-leads:
             description: Team of Pipeline Working Group leads
             members:
-            - animeshsingh
             - chensun
+            - droctothorpe
             - james-jwu
-            - IronPan
-            - zijianjoy
-            privacy: closed
+            - HumairAK
+            - mprahl
+            - zazulam
+            privacy: open
```

Comment:
Can you also create PR to keep privacy: closed for this GitHub team @anishasthana ?
It looks like our CI doesn't update the team: https://github.com/orgs/kubeflow/teams/wg-pipeline-leads

Reply:
I opened https://github.com/kubeflow/internal-acls/pull/810